cli/src/plugins/javadoc/index.ts

```diff
@@ -122,6 +122,20 @@ async function processJson(
   await Promise.all(promises);
 }
 
+function getTitle(doc: ParsedClassDoc): string {
+  var type = ""
+  if (doc.isInterface) {
+    type = "Interface"
+  } else if (doc.isClass) {
+    type = "Class"
+  } else if (doc.isEnum) {
+    type = "Enum"
+  } else if (doc.isException) {
+    type = "Exception"
+  }
```

Comment:
What if it's none of those things? (Not sure how that would be possible, but maybe?)

Reply:
Then we just use an empty type and end up with the document name instead. However, there is a preceding space right now -- i'll trim that.